### ISSUE TITLE
API documentation for non-spreadsheet runs

### ISSUE THREAD

**Issue by pbeaucage** [NSLS-II-SST/rsoxs_scans#20]
It is frequently desired or required to run RSoXS scans outside the context of a spreadsheet, for example:

- when using user-supplied holders in the TEM port
- (hypothetical) when using user-supplied sample holders in the traditional positioning port
- performing alignment-type scans that are outside the envisioned scope of the `rsoxs_scans` package, for example grazing incidence, CDSAXS-style rotation series, HDR exposure series for DAFS-style RSoXS measurements, etc. 

This is presently medium-level supported by the existing scan scheme, `full_carbon_scan`, etc.

Is API-level access to this package possible?
If so, can some documentation be generated of how `rsoxs_scans` is used to _interactively_ conduct scans from an ipython session or similar, rather than through spreadsheets?
If not, will API support for the legacy RSoXS scans be maintained indefinitely?

**Comment by EliotGann** [comment]
yes, there are plans which wrap these in the rsoxs module.  do_rsoxs, do_spiral, do_nexafs. each with kwargs of the column names should work just the same as from the spreadsheet. the legacy RSoXS scans have been removed and there is no plan to allow them back currently.

**Comment by EliotGann** [comment]
https://github.com/NSLS-II-SST/rsoxs/blob/new-scan-scheme/rsoxs/Functions/rsoxs_plans.py is the integration of the beamline code to this repo

**Comment by EliotGann** [comment]
I understand that this is essentially blueskyifying bluesky ...  we need a message bus to run our message bus to run our message bus ...
